microsoft-edge/webdriver-chromium/capabilities-edge-options.md

```diff
@@ -48,7 +48,7 @@ Most Microsoft Edge-specific capabilities are exposed through the `EdgeOptions`
 
 | Capability | Type | Default value | Details |
 |:--- |:--- |:--- |:--- |
-| args | list of strings |  | List of command-line arguments to use when starting Microsoft Edge.  Arguments with an associated value should be separated by a `=` sign (for example, `['start-maximized', 'user-data-dir=/tmp/temp_profile']`). |
+| args | list of strings |  | List of command-line arguments to pass to the Microsoft Edge process on launch. Arguments with an associated value should be separated by a `=` sign (for example, `['start-maximized', 'user-data-dir=/tmp/temp_profile']`). If you're launching a WebView2 app, then these arguments are passed to your app instead of the underlying Microsoft Edge browser process. To pass arguments to the browser process when launching a WebView2 app, use [webviewOptions.additionalBrowserArguments](#webviewoptions-object) instead. |

```

Comment:
```suggestion
| args | list of strings |  | List of command-line arguments to pass to the Microsoft Edge process on launch. Arguments with an associated value should be separated by an `=` sign (for example, `['start-maximized', 'user-data-dir=/tmp/temp_profile']`). If you're launching a WebView2 app, then these arguments are passed to your app instead of the underlying Microsoft Edge browser process. To pass arguments to the browser process when launching a WebView2 app, use [webviewOptions.additionalBrowserArguments](#webviewoptions-object) instead. |

```